backend/adapter_processor/models.py

```diff
@@ -93,6 +98,28 @@ class Meta:
             ),
         ]
 
+    def get_adapter_meta_data(self) -> Any:
+        encryption_secret: str = settings.ENCRYPTION_KEY
+        f: Fernet = Fernet(encryption_secret.encode("utf-8"))
+
+        adapter_metadata = json.loads(
+            f.decrypt(bytes(self.adapter_metadata_b).decode("utf-8"))
+        )
+        return adapter_metadata
+
+    def get_context_window_size(self) -> int:
+
```

Comment:
NIT: please remove un-necessry empty lines. Here and 118

Reply:
addressed